packages/commonjs/README.md

```diff
@@ -182,13 +182,44 @@ This is in line with how other bundlers handle this situation and is also the mo
 
 For these situations, you can change Rollup's behaviour either globally or per module. To change it globally, set the `requireReturnsDefault` option to one of the following values:
 
-- `false`: This is the default, requiring an ES module returns its namespace. For external dependencies when using `esmExternals: true`, no additional interop code is generated:
+- `false`: This is the default, requiring an ES module returns its namespace. This is the only option that will also add a marker `__esModule: true` to the namespace to support interop patterns in CommonJS modules that are transpiled ES modules.
 
   ```js
   // input
   const dep = require('dep');
   console.log(dep);
 
+  // output
+  import * as dep$1 from 'dep';
+
+  function getAugmentedNamespace(n) {
```

Comment:
Perhaps this function could have an explicit opt-out if `__esModule` is already defined? Would that be a slight perf improvement in some cases?

Reply:
Definitely, added!